fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+    # Draft Orders API 
+      
+      A [Draft Order](https://help.shopify.com/manual/orders/create-orders) represents the status of an order before the order is set to `completed` status. Using the Draft Orders API you can create a draft order and transition it to a completed order with great flexibility.  The following flows are supported:
+      
+      * Create a draft order that calculates taxes and totals but accept payment from the customer outside of Shopify.
+      * Create a draft order (with taxes and totals) and send your customer a web checkout link to complete purchase and transition to a completed order.  
+      * Use Draft Orders API in conjunction with the [Checkouts API](https://help.shopify.com/api/tutorials/sell-through-the-checkout-api) by creating a Checkout object based on a draft order (Sales Channel SDK only).  This enables a highly customizable experience for channels.
+      
+     ## About Draft Orders
+    
+     When using the [Orders API](https://help.shopify.com/api/reference/order) to create orders, no payment information is calculated and no transaction is performed.  With the Draft Orders API, on the other hand, you can create orders on which shipping, taxes and all total amounts are calculated by Shopify.  This enables a wide range of use cases including the following:
+     
+      * Create new orders and enter credit card payments in Shopify for sales you've made by phone, in person, or elsewhere.
+      * Send invoices to customers to pay with a secure checkout link
+      * Use custom items to represent additional costs or products that aren't displayed in your inventory.
+      * Manually re-create mistaken orders from any of your active sales channels
+      * Sell products at discount or wholesale rates
+      * Take pre-orders.
+
+
+schemes:
+  - https
+  
+host: your-store-domain.myshopify.com
+
+tags:
+  - name: Get
+    description: Retrieve draft orders.
+  - name: Create
+    description: Create draft orders.
+  - name: Delete
+    description: Delete draft orders
+  - name: Update
+    description: Update draft orders.
+  - name: To Be Implemented
+    description: Operations not ready for Alpha launch.
+  - name: Response Codes
+    description: |
+      Draft order endpoints will use HTTP status codes to differentiate between draft orders with pending operations (using a 202 Accepted) and draft orders which are successfully created (201 Created) or updated (200 OK).
+
+
+      In addition to the HTTP status codes, responses will also use Location and Retry-After headers to specify the polling URI endpoint and polling frequency respectively.
+      
+      
+      The response will contain the partially calculated draft order and any calculated attributes dependant on stepwise operations will be null/empty until available.
+      
+      For long running operations, clients will be responsible for polling the endpoint specified in the Location header until a 200 OK response is returned.
```

Comment:
* "long running applications" -> "long-running applications"
* "will be responsible" -> "are responsible"